### ISSUE TITLE
Wiki links syntax support?

### ISSUE THREAD

**Issue by k8ieone** [toolstack/Folio#152]
Supporting `[[Some Note]]` syntax to link to a note called *Some Note* would bring Folio one step closer to be able to replace tools like Obsidian (and even bring compatibility with their implementation).

I'm not sure if this would be out of scope tho.

**Comment by toolstack** [comment]
It's not part of the markdown spec and you can already link to other notes using the url syntax, so certainly not a high priority item.

**Comment by k8ieone** [comment]
Ah, didn't know that. That's also great, thanks for letting me know.

**Comment by toolstack** [comment]
> Folio could interpret that as a potentially missing note in the same folder and offer to create it on the fly.

I don't think that would be a good UI, it would create a lot of false positives due to typo's etc and only save a single click anyway in the best case scenario.
